.github/workflows/native_doc_dartifier.yaml

```diff
@@ -0,0 +1,63 @@
+name: native_doc_dartifier
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - ".github/workflows/native_doc_dartifier.yaml"
+      - "pkgs/native_doc_dartifier/**"
+  pull_request:
+    branches: [main]
+    paths:
+      - ".github/workflows/native_doc_dartifier.yaml"
+      - "pkgs/native_doc_dartifier/**"
+
+env:
+  PUB_ENVIRONMENT: bot.github
+
+jobs:
+  analyze:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: pkgs/native_doc_dartifier/
+    strategy:
+      fail-fast: false
+      matrix:
+        sdk: [stable]
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
Please pin all of the exact commit versions of these actions similar to https://github.com/dart-lang/native/blob/main/.github/workflows/jnigen.yaml#L49